README.md

```diff
@@ -16,12 +16,15 @@ You'll also need DocFX installed as a command-line tool.  This can be accomplish
 Alternatively, you can download the latest release of `docfx.zip` from [DocFX's GitHub Releases](https://github.com/dotnet/docfx/releases), extract it to a local directory, and add that directory to `PATH` so you can run it anywhere.
 
 ## Setting Up the DNN Docs Project
-After installing DocFX, the next step is to clone this repository from GitHub to your local machine.  This will create a copy of the DNNDocs Git repository (its directories and folders) on your local machine so that you can work with them there.
+After installing DocFX, the next step is to fork this repository from GitHub to your own GitHub repository in order to give you write access on your own copy (if you don't have a GitHub account, you can create a free one).
+Simply click the Fork button above to get started.
+
+After your own fork is created, you need to copy the repository into your local machine, this is called cloning. This will create a copy of the DNNDocs Git repository (its directories and folders) on your local machine so that you can work with them there.
 
 The following command will clone this repository to your working directory, so you may wish to `cd` to an appropriate directory first.
```

Comment:
The following command will clone the forked repository to your working directory, so you may wish to `cd` to an appropriate directory first.